src/lib/endpoints/channels.ts

```diff
@@ -68,11 +70,8 @@ export class ChannelsEndpoint extends Endpoint {
 
 	public async list(options: ListOptions = {}): Promise<Channel[]> {
 		const params: HttpClientParams = {}
-		if (options.type) {
-			params.type = options.type
-		}
-		if (options.subscriberId) {
-			params.subscriberId = options.subscriberId
+		if (typeof(options.includeReadOnly) === 'boolean') {
```

Comment:
I thought Typescript would already ensure this at compile time. Is there a runtime case where a simple truthy check is wrong?

Reply:
Since the library is callable by JavaScript we should probably be being more careful about things like this but that's not the reason I included this anyway. It's because `options.includeReadOnly` is optional, i.e. actually `boolean | undefined` and I'm leaving the parameter off when the user doesn't specify it.